model/perf.go

```diff
@@ -80,8 +80,11 @@ func CreatePerformanceResult(info PerformanceResultInfo, source []ArtifactInfo,
 			Stats:       append([]PerfRollupValue{}, rollups...),
 			ProcessedAt: createdAt,
 		},
-		populated:    true,
-		ChangePoints: []ChangePoint{},
+		populated: true,
+		Analysis: PerfAnalysis{
+			ChangePoints: []ChangePoint{},
+			ProcessedAt:  time.Time{},
```

Comment:
I believe that the zero/default initlizations of these values are fine, and that you don't need to specify any of these values at all.